setup.sql

```diff
@@ -34,8 +34,12 @@ CREATE TABLE cart_item (
 --wip
 CREATE TABLE orders (
 	id SERIAL PRIMARY KEY,
-	items JSON, -- will be changed later to something that can be an array of order_items
-	subtotal DECIMAL(10,2) -- for development use only, will be removed later
+	items JSON[], -- revisit for possible junction table implementation
+	subtotal DECIMAL(10,2),
+	discount DECIMAL(10,2)
+	total DECIMAL(10,2),
+	payment_status BOOLEAN,
+	date_ordered DATE NOT NULL,
```

Comment:
remove the comma in the end 
(also should probably comment this out or remove the NOT NULL part so earlier functionality won't break
though if you are going to comment this out, make sure to remove the comma in payment_status)